cmd/state-svc/main.go

```diff
@@ -185,6 +185,7 @@ func runForeground(cfg *config.Instance, an *anaSync.Client, auth *authenticatio
 
 	logFile := logging.FilePath()
 	logging.Debug("Logging to %q", logFile)
+	go logging.RotateLogTimer()
```

Comment:
nit: It's usually recommended for packages to abstract away goroutines. So `logging.RotateLogTimer()` would be non-blocking and manage the goroutine and whatever backpressure (i.e. errors, channel outputs) that is required, which is none in this case (IIUC).

Reply:
I agree. Thanks for pointing this out.